doc/api.yml

```diff
@@ -1832,6 +1832,8 @@ components:
           type: string
         nic:
           type: string
+        profile_picture:
+          type: string
```

Comment:
unclear,
What kind of data will be stored here ? a link ? a filename ?

Reply:
I add some description to describe the attribute